### DISCUSSION TITLE
numero is not defined na função Matematica.maior_numero(numero1,numero2)

### DISCUSSION THREAD

**Discussion by rcepgs2022** [dgadelha/Portugol-Webstudio#246] [Dúvidas / Perguntas] [answered]
Por que o código abaixo funciona normalmente na versão Portugol Studio, mas não funciona na versão Portugol Web Studio?

```
programa
{
	inclua biblioteca Matematica --> mat
	
	funcao inicio()
	{
		real numero1
		real numero2
		real maior

		escreva("Informe o primeiro número: ")
		leia(numero1)

		escreva("Informe o segundo número: ")
		leia(numero2)

		// Obtém o maior dos dois números
		maior = mat.maior_numero(numero1, numero2)
		escreva("\nO maior número é: ", maior, "\n")
	}
}
```

![maior_numero_portugol_studio](https://github.com/dgadelha/Portugol-Webstudio/assets/119897018/29fe328e-a28b-4376-b61f-e18371098ef4)

![maior_numero_portugol_webstudio](https://github.com/dgadelha/Portugol-Webstudio/assets/119897018/102b85d0-225b-4e29-8e16-ae720dc550be)

**Comment by dgadelha** [comment, marked as answer]
Obrigado pelo report! Criei a issue #258 e corrigi o problema no commit 442fce494caf27eccc0e06194f37b26a6cd03072.
